File: cc/006.cc
class Solution {
public:
    int minNumberInRotateArray(vector<int> rotateArray) {
        if (rotateArray.empty()) return 0;
        for (int i = 0; i < rotateArray.size() - 2; i++)
            if (rotateArray[i] > rotateArray[i+1])
                return rotateArray[i+1];
        return 0;
    }
};

File: python/006.py
# -*- coding: utf-8 -*-
# /usr/bin/python3
# Creator: Hemajun
# Description: 006


class Solution:
    def minNumberInRotateArray(self, rotateArray):
        length = len(rotateArray)
        if length != 0:
            for i in range(length - 2):
                if rotateArray[i] > rotateArray[i+1]:
                    return rotateArray[i+1]
        return 0
